hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4057,6 +4057,12 @@ public static boolean isAclEnabled(Configuration conf) {
   public static final String FS_NODE_LABELS_STORE_ROOT_DIR = NODE_LABELS_PREFIX
       + "fs-store.root-dir";
 
+  /** Node label file replication */
+  public static final String FS_NODE_LABELS_FILE_REPLICATION =
+      NODE_LABELS_PREFIX + "file.replication";
+
```

Comment:
Remove this blank line, so that related configs can be group by.

Reply:
Removed blank line